src/Module/Settings/Display.php

```diff
@@ -297,19 +330,19 @@ private function getAvailableTimelines(int $uid, bool $only_network = false): ar
 		$timelines = [];
 
 		foreach ($this->timeline->getNetworkFeeds('') as $channel) {
-			$timelines[$channel->code] = $this->t('%s: %s', $channel->label, $channel->description);
+			$timelines[$channel->code] = $channel;
```

Comment:
No need for this anymore.
```suggestion
			$timelines[] = $channel;
```

Reply:
There is a need for this: https://github.com/friendica/friendica/pull/13448/files#diff-7ba9efa85e016f31a6289519a3e365556860869930e8963479892623ee8f15bbR243

Reply:
Good catch, I addressed this in https://github.com/friendica/friendica/pull/13448#discussion_r1342177455

Reply:
Done